README-trinity.md

```diff
@@ -0,0 +1,8 @@
+# Trinity
```

Comment:
Not really sure if we need a `README-trinity.md` on the root level (unless it serves a special purpose that I might be overseeing). I think the `README.md` on the root level serves as explainer for the entire repo so even in a mono repository scenario there would only be one top level `README.md`.

Reply:
I assume it's just an intermediate step toward what would be the `README.md` file in the new `trinity` repo.

Reply:
This was a leftover from a previous design.  Removing for now.